docs/src/lca.md

```diff
@@ -1,6 +1,6 @@
 # Leaky Competing Accumulator
 
-The Leaky Competing Accumulator (LCA; Usher & McClelland, 2001) is a sequential sampling model in which evidence for options races independently. The LBA makes an additional simplification that evidence accumulates in a linear and ballistic fashion, meaning there is no intra-trial noise. Instead, evidence accumulates deterministically and linearly until it hits the threshold.
+The Leaky Competing Accumulator (LCA; Usher & McClelland, 2001) is a sequential sampling model in which evidence for options races independently. the LCA is similar to the Linear Ballistic Accumulator (LBA), but additionally assumes an intra-trial noise in the form of evidence accumulation leakage (in contrast, the The LBA assumes that evidence accumulates in a ballistic fashion, i.e., linearly and deterministically until it hits the threshold).
```

Comment:
"the The" in the parentheses

Reply:
Noise and leakage are separate concepts. I would say something like:

"intra-trial noise and leakage (in contrast, the The LBA assumes that evidence accumulates in a ballistic fashion, i.e., linearly and deterministically until it hits the threshold)